cli/tests/test_config_command.rs

```diff
@@ -108,7 +108,7 @@ fn test_config_list_inline_table() {
     );
     let stdout = test_env.jj_cmd_success(test_env.env_root(), &["config", "list", "test-table"]);
     insta::assert_snapshot!(stdout, @r###"
-    test-table=[{x=1}, {y=["z"]}]
+    test-table=[{ x = 1 }, { y = ["z"] }]
```

Comment:
You could use the test from https://github.com/martinvonz/jj/pull/3731/files#diff-4256ae2574c92c8fe81a570c6f4df6b91d19cbe3465cb1b10560986407c8ca83.